gammapy/makers/utils.py

```diff
@@ -671,6 +672,80 @@ def make_effective_livetime_map(observations, geom, offset_max=None):
     return livetime
 
 
+def get_effective_livetime(
+    datastore, position, radius=None, border=None, en_edges=None
+):
+    """Compute the effective observation time at a given sky position.
+
+    This function computes the total livetime over observations containing a region of interest. Given the fact
+    that the instrument responses change within the field of view for each observation, this function computes
+    precisely the acceptance-corrected livetime map.
+
+    Parameters
+    ----------
+    datastore: `~gammapy.data.DataStore`
+        Data store of a data release.
+    position : `~astropy.coordinates.SkyCoord`
+        Central sky position.
+    radius : `~astropy.coordinates.Angle`, optional.
+        Radius of the region of interest.
+        If None, a `radius` of 0.1 deg is used. Default is None.
+    border : `~astropy.coordinates.Angle`, optional.
+        Additional edge summed to the region of interest used for the selection of the observations.The maximum offset FoV. Default is None.
+        If None, a `border` of 6 deg is used. Default is None.
+    en_edges : list of `~astropy.units.Quantity`
+        Edges of true energy bins. Default is None.
+
+    Returns
+    -------
+    livetime_maps : `~gammapy.maps.Map` or list of `~gammapy.maps.Map`
+        Effective livetime map(s).
+    selected_ids : list of `str`
+        List of selected observation IDs.
+
+    # Examples
+    # --------
+    # ToDo: add the tutorial link after
+    """
+
+    border = Angle(6, "deg") if border is None else border
+    selection = dict(
+        type="sky_circle",
+        frame="icrs",
+        lon=position.ra,
+        lat=position.dec,
+        radius=Angle(0.1, "deg") if radius is None else radius,
+        border=border,
+    )
+    selected_obs_table = datastore.obs_table.select_observations(selection)
```

Comment:
what about `required_irf`? This won't work for a pointlike data distribution.

Reply:
Good point... we need to check whether the previously coded functions works also (which was not anticipated in the past).

Reply:
The core Gammapy function `make_effective_livetime_map` is indeed even not tested for point-like IRFs...
Should I make a separate issue?
For safety, I change the way to get the list of obs_id...